RandBLAS/sparse.hh

```diff
@@ -149,13 +221,16 @@ template <typename SKOP>
 static bool has_fixed_nnz_per_col(
     SKOP const& S0
 ) {
-    if (S0.dist.family == SparseDistName::SASO) {
+    if (S0.dist.family == SparsityPattern::SASO) {
         return S0.dist.n_rows < S0.dist.n_cols;
     } else {
         return S0.dist.n_cols < S0.dist.n_rows;
     }
 }
 
+// =============================================================================
+/// WARNING: this function is not part of the public API.
+///
```

Comment:
I think we still should document functions/classes that are not in the public API so that anyone on the team can easily use them. We can hide stuff from doxygen with the `@cond` and `@endcond` directives

Reply:
Yeah I agree. I'll make those changes in the next PR.